static/app/components/events/interfaces/crashContent/exception/utils.spec.tsx

```diff
@@ -0,0 +1,121 @@
+import {render} from 'sentry-test/reactTestingLibrary';
+
+import {Linkify} from 'sentry/components/events/interfaces/crashContent/exception/utils';
+
+describe('Linkify()', function () {
+  it('does not alter text that does not contain urls', function () {
+    const text = 'This is not a link';
+    const {container, getByText} = render(<Linkify exceptionText={text} />);
```

Comment:
Instead of using the returned `getByText` and other matchers, it's preferred to use `screen.getByText` which is imported at the top of the file